regulations/templates/regulations/section.html

```diff
@@ -19,13 +19,13 @@
 
 <div class="box">
 
-    <aside class="left-sidebar" data-state="expanded">
+    <aside class="left-sidebar" data-state="expanded" data-collapse-target="left-sidebar">
         <div class="toc-controls">
-            <button data-set-state="collapsed">
+            <button data-set-state="collapsed" data-collapse-target="left-sidebar">
```

Comment:
Is there a name more about state than about the collapse? Something like `data-state-name="left-sidebar"` that describes the things effected by that particular state.

Reply:
Fixed in 98fa966